django/common/analyzer/condition.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         self.value = value
 
     def check(self, data: Dict[DataDictKey, DataDictValue]):
-        data_value = data[(CONDITION_FLAG, (self.sql_column.table, self.sql_column.column))]
+        data_value = data[(CONDITION_FLAG, (self.sql_column.owner, self.sql_column.table, self.sql_column.column))]
```

Comment:
I think it should be `data_value = data[(CONDITION_FLAG, (self.sql_column.table_name(), self.sql_column.column))]` so that the squash rules work